tests/components/myuplink/snapshots/test_init.ambr

```diff
@@ -0,0 +1,97 @@
+    'labels': set({
+    }),
+    'manufacturer': 'Jäspi',
+    'model': 'Tehowatti Air',
+    'model_id': None,
+    'name': 'Gotham City',
+    'name_by_user': None,
+    'primary_config_entry': <ANY>,
+    'serial_number': '10001',
+    'suggested_area': None,
+    'sw_version': '9682R7A',
+    'via_device_id': None,
+  })
+# ---
+# name: test_device_info[{\n  "id": "batman-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff",\n  "connectionState": "Connected",\n  "firmware": {\n    "currentFwVersion": "9682R7B",\n    "desiredFwVersion": "9682R7B"\n  },\n  "product": {\n    "serialNumber": "10002",\n    "name": "F730 CU 3x400V"\n  },\n  "availableFeatures": {\n    "settings": true,\n    "reboot": true,\n    "forcesync": true,\n    "forceUpdate": false,\n    "requestUpdate": false,\n    "resetAlarm": true,\n    "triggerEvent": true,\n    "getMenu": false,\n    "getMenuChain": false,\n    "getGuideQuestion": false,\n    "sendHaystack": true,\n    "setSmartMode": false,\n    "setAidMode": true,\n    "getZones": false,\n    "processIntent": false,\n    "boostHotWater": true,\n    "boostVentilation": true,\n    "getScheduleConfig": false,\n    "getScheduleModes": false,\n    "getScheduleWeekly": false,\n    "getScheduleVacation": false,\n    "setScheduleModes": false,\n    "setScheduleWeekly": false,\n    "setScheduleOverride": false,\n    "setScheduleVacation": false,\n    "setVentilationMode": false\n  }\n}\n-batman-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff-{\n  "page": 1,\n  "itemsPerPage": 10,\n  "numItems": 3,\n  "systems": [\n    {\n      "systemId": "123456-7890-1234",\n      "name": "Gotham City",\n      "securityLevel": "admin",\n      "hasAlarm": false,\n      "country": "Sweden",\n      "devices": [\n        {\n          "id": "alfred-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff",\n          "connectionState": "Connected",\n          "currentFwVersion": "9682R7A",\n          "product": {\n            "serialNumber": "10001",\n            "name": "Tehowatti Air"\n          }\n        }\n      ]\n    },\n    {\n      "systemId": "123456-7890-1234",\n      "name": "Batcave",\n      "securityLevel": "admin",\n      "hasAlarm": false,\n      "country": "Sweden",\n      "devices": [\n        {\n          "id": "batman-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff",\n          "connectionState": "Connected",\n          "currentFwVersion": "9682R7B",\n          "product": {\n            "serialNumber": "10002",\n            "name": "F730 CU 3x400V"\n          }\n        }\n      ]\n    },\n    {\n      "systemId": "123456-7890-1234",\n      "name": "Duckburg",\n      "securityLevel": "admin",\n      "hasAlarm": false,\n      "country": "Sweden",\n      "devices": [\n        {\n          "id": "robin-r-1234-20240201-123456-aa-bb-cc-dd-ee-ff",\n          "connectionState": "Connected",\n          "currentFwVersion": "9682R7C",\n          "product": {\n            "serialNumber": "10003",\n            "name": "SM0 20"\n          }\n        }\n      ]\n    }\n  ]\n}\n]
```

Comment:
Can we customize the name here of the fixture? Instead of passing the contents of the fixture?

../Frenck

Reply:
@frenck This was more difficult (for me) than I thought. There are interactions between parameterize and snapshot:
```
snapshot(name="custom_name") 
results in:
 f"{name_of_test} [{parameter1}][{parameter2}][{custom_name}]"
i.e.
test_device_info[the_fixture_content][the_device_id][custom_name]
```
I have tried to get around the default parts of the name without any success.

Reply:
It seems that the parameters from @parametrize are ALWAYS added to the name string. I have even tried to modify HomeAssistantSnapshotExtension without success.